packages/sui-hoc/src/withOpenToggle.js

```diff
@@ -1,15 +1,21 @@
 import React, {Component} from 'react'
+import PropTypes from 'prop-types'
 
 export default BaseComponent => {
   const displayName = BaseComponent.displayName
 
   return class WithOpenToggle extends Component {
     state = {
-      isOpen: false
+      isOpen: !!this.props.isOpen
```

Comment:
https://medium.com/dev-genius/vs-boolean-cd502fbd6101

I personally prefer Boolean(whatever)

Reply:
I suggested `!!` because IMHO looks shorter and equally understandable for non-complex expressions, but as the article states (good one BTW) both are nice options!